### PR TITLE
Fixing storing to hdf

### PR DESCRIPTION
**Pull request by KasukabeDefenceForce**
### :pencil: Description

**Type:** :beetle: `bugfix`

This PR aims at fixing [2622](https://github.com/tardis-sn/tardis/issues/2622).

Also, link issues affected by this pull request by using the keywords: `close`, `closes`, `closed`, `fix`, `fixes`, `fixed`, `resolve`, `resolves` or `resolved`. 

### :pushpin: Resources

Examples, notebooks, and links to useful references.

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [ ] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [ ] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: tardis/model/parse_input.py [hunk 1]
```diff
@@ -417,7 +417,7 @@ def parse_abundance_csvy(
         _, mass_fraction, isotope_mass_fraction = parse_csv_abundances(
             csvy_model_data
         )
-        mass_fraction = mass_fraction.loc[:, 1:]
+        # mass_fraction = mass_fraction.loc[:, 1:]
```
Comment (KasukabeDefenceForce) [inline tardis/model/parse_input.py:420]: Why are we removing the first column?

File: tardis/io/model_reader.py [hunk 2]
```diff
@@ -553,53 +553,58 @@ def transport_to_dict(transport):
...
-        "input_nu": transport.input_nu,
-        "input_r_cgs": transport.input_r,
-        "j_blue_estimator": transport.j_blue_estimator,
-        "j_estimator": transport.j_estimator,
-        "last_interaction_in_nu": transport.last_interaction_in_nu,
-        "last_interaction_type": transport.last_interaction_type,
-        "last_line_interaction_in_id": transport.last_line_interaction_in_id,
-        "last_line_interaction_out_id": transport.last_line_interaction_out_id,
-        "last_line_interaction_shell_id": transport.last_line_interaction_shell_id,
+        # "input_energy": transport.input_energy,
```
Comment (KasukabeDefenceForce) [inline tardis/io/model_reader.py:560]: What should be done with these keys?

File: tardis/io/model_reader.py [hunk 3]
```diff
@@ -553,53 +553,58 @@ def transport_to_dict(transport):
...
-        "nu_bar_estimator": transport.nu_bar_estimator,
-        "photo_ion_estimator": transport.photo_ion_estimator,
-        "photo_ion_estimator_statistics": transport.photo_ion_estimator_statistics,
-        "r_inner": transport.r_inner_cgs,
-        "r_outer": transport.r_outer_cgs,
+        "nu_bar_estimator": transport.transport_state.radfield_mc_estimators.nu_bar_estimator,
+        "photo_ion_estimator": transport.transport_state.radfield_mc_estimators.photo_ion_estimator,
+        "photo_ion_estimator_statistics": transport.transport_state.radfield_mc_estimators.photo_ion_estimator_statistics,
+        "r_inner": transport.transport_state.geometry_state.r_inner,
+        "r_outer": transport.transport_state.geometry_state.r_outer,
```
Comment (KasukabeDefenceForce) [inline tardis/io/model_reader.py:576]: Are the paths correct for these keys? as I have removed 'cgs' from the end

File: tardis/io/model_reader.py (hunk 2 above)
Comment (Knights-Templars) [inline tardis/io/model_reader.py:560]: Look at the code from Last June.

**Comment by andrewfullard** [comment]
is this still actually a problem?

**Pull request closed without merging**